### PR TITLE
Added configs for initial deployment of carma-cdw

### PR DESCRIPTION
**Pull request by chaupt-va**
- Updated Jenkinsfile
- Added the required `conf` and `yaml` files

### CODE REVIEW

File: ingress.tests [hunk 1]
```diff
@@ -25,6 +25,9 @@ carma /api/carma/actuator/info
 carma /api/carma/vetSearch
 carma /api/carma/mviSearch
 carma /api/carma/eeSearch
+# Carma Cdw
+carma-cdw /carma-cdw/v0/actuator/info
```
Comment (jhulbert-va) [inline ingress.tests]: Are there any paths other than the actuator ones?

File: products/carma-cdw.conf [hunk 2]
```diff
@@ -0,0 +1,9 @@
+DU_ARTIFACT=vecms-carma-cdw
```
Comment (jhulbert-va) [inline products/carma-cdw.conf]: `vecms-carma-cdw-deployment`

File: products/carma-cdw.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+---
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: carma-cdw-resource-quota
+  namespace: $DU_NAMESPACE
+spec:
+  hard:
+    limits.cpu: "6000m"
+    limits.memory: "16G"
```
Comment (jhulbert-va) [inline products/carma-cdw.yaml]: These limits are a huge jump from your application limits. Your deployment config has replicas for `carma-cdw` set to 1 with limits of `600m` for CPU and `2G` for memory. These limits shouldn't be too much more than twice that.

File: ingress.tests (hunk 1 above)
Comment (chaupt-va) [reply ingress.tests]: No, not at the moment. We've only created a skeleton for the app.

**Comment by dlennon-va** [comment]
> I'm good with you trying to deploy it from the branch in Jenkins.

Successfully deployed to QA from this branch.

File: Jenkinsfile [hunk 4]
```diff
@@ -7,8 +7,9 @@
 def products() {
   products = [:]
   products["none"] = ["health_apis_jenkins"]
-  products["carma"] = ["health_apis_jenkins"]
-  products["carma-fms-connector"] = ["health_apis_jenkins"]
+  products["carma"] = ["health_apis_jenkins", "arcadian_achievements"]
+  products["carma-fms-connector"] = ["health_apis_jenkins", "arcadian_achievements"]
+  products["carma-cdw"] = ["health_apis_jenkins", "arcadian_achievements"]
```
Comment (bschofield-va) [inline Jenkinsfile]: c before f

File: products/carma-cdw.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+              servicePort: 80
+---
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: carma-cdw-resource-quota
+  namespace: $DU_NAMESPACE
+spec:
+  hard:
+    limits.cpu: "1300m"
```
Comment (bschofield-va) [inline products/carma-cdw.yaml]: were these limits sorted out by @mramirez-va

Comment (mramirez-va) [reply products/carma-cdw.yaml]: @chaupt-va https://github.com/department-of-veterans-affairs/health-apis-deployer/blob/qa/deployment-unit.md#defining-resource-requestslimits

**Pull request merged**